migrations/20241011154110_create_affected_packages.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS affected_packages (
+  affected_package_id uuid primary key,
+  advisory_id uuid references security_advisories,
+  package_id uuid references packages not null,
+  cvss text not null,
+  introduced_version uuid references releases not null,
+  fixed_version uuid references releases,
+  architecture text[],
+  os text[],
+  declarations text[][]
```

Comment:
I thin it is somehow a weird choice to have an additional primary key in this table since it mainly represents a relation between three other tables ~ but I guess it makes sense for consistency, eh?

Reply:
@MangoIV There is only one primary key, it's the UUID that we control.